src/test/java/org/kohsuke/github/AppTest.java

```diff
@@ -584,6 +584,8 @@ public void testCommitShortInfo() throws Exception {
         GHCommit commit = r.getCommit("86a2e245aa6d71d54923655066049d9e21a15f23");
         assertEquals(commit.getCommitShortInfo().getAuthor().getName(), "Kohsuke Kawaguchi");
         assertEquals(commit.getCommitShortInfo().getMessage(), "doc");
+        assertFalse(commit.getCommitShortInfo().getVerification().getVerified());
+        assertEquals(commit.getCommitShortInfo().getVerification().getReason(), GHReason.UNSIGNED.get());
```

Comment:
As noted above, we need tests for at least a few more `GHReason` values (specifically multi-word values).